GettingStarted.md

```diff
@@ -0,0 +1,181 @@
+# Getting started with TEE development
+The instructions below will help you get your development environment configured so that you can build, test and debug Team Explorer Everywhere.
+
+## Clone and configure the repository
+Use the Git tool of your choice to clone the repository into a local path.
+For example, you could use git.exe from a Windows console window:
+```
+mkdir c:\repos
+pushd c:\repos
+git clone https://github.com/Microsoft/team-explorer-everywhere.git
+```
+
+We keep LF line-ending on the server. Please set the `core.safecrlf` git config property to `true`.
+```
+git config --local core.safecrlf true
+```
+
+## Install development tools
+
+### Install Java
+
+Two versions of the Java Development Kit are needed: we use Java SE 6 as the minimal supported Java execution environment and to build the release kit, as well as Java SE 8 for running Eclipse.
+
+1. Download and install Java SE 6:
+  * [Windows and Linux: Java Development Kit 6u45](http://www.oracle.com/technetwork/java/javase/downloads/java-archive-downloads-javase6-419409.html#jdk-6u45-oth-JPR)
+  * [Mac: Apple Java 1.6.0_65, a.k.a. Java for OS X 2015-001](https://support.apple.com/kb/DL1572?locale=en_US)
+1. Set the JAVA_HOME environment variable to point to the install, e.g
+ * (Windows) `SET JAVA_HOME=C:\Program Files\Java\jdk1.6.0_45`
+ * (Linux) `JAVA_HOME=~/dev/java/jdk1.6.0_45`
```

Comment:
> Shouldn't we use the export command to set environment variables on Linux and Mac?

The thought had crossed my mind, especially since this document is intended to be for one-time instructions and just setting or exporting at the console/terminal won't make it persistent.  Since the intent is to have changes to the environment be persistent, I've replaced the examples with a note to that effect.  If we find enough users/developers tripping up on this part, we can always introduce batch and bash scripts that configure the current console/terminal appropriately, kind of like the "Visual Studio Command Prompt" batch file did.